app/app.go

```diff
@@ -666,6 +643,40 @@ func NewStrideApp(
 		app.AccountKeeper, app.BankKeeper, app.DistrKeeper, app.StakingKeeper,
 	)
 
+	// Add wasm keeper
+	// QUESTION: do we need any custom opts, stargate, or ICS20 wasm hooks? Assuming no to all
+	wasmCapabilities := "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4"
+	wasmDir := filepath.Join(homePath, "wasm")
+	wasmConfig, err := wasm.ReadWasmConfig(appOpts)
+	if err != nil {
+		panic(fmt.Sprintf("error while reading wasm config: %s", err))
+	}
+
+	scopedWasmKeeper := app.CapabilityKeeper.ScopeToModule(wasmtypes.ModuleName)
+
+	app.WasmKeeper = wasmkeeper.NewKeeper(
+		appCodec,
+		keys[wasmtypes.StoreKey],
+		app.AccountKeeper,
+		app.BankKeeper,
+		app.StakingKeeper,
+		distrkeeper.NewQuerier(app.DistrKeeper),
+		app.IBCKeeper.ChannelKeeper, // ICS4Wrapper
+		app.IBCKeeper.ChannelKeeper,
+		&app.IBCKeeper.PortKeeper,
+		scopedWasmKeeper,
+		app.TransferKeeper,
+		app.MsgServiceRouter(),
+		app.GRPCQueryRouter(),
+		wasmDir,
+		wasmConfig,
+		wasmCapabilities,
+		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+		wasmOpts...,
```

Comment:
@p0mvn do you know if we need to include the [stargate wasm options](https://github.com/osmosis-labs/osmosis/blob/main/app/keepers/keepers.go#L483-L484)? 

For context, we want a minimum installation that would allow deploying hyperlane.

My assumption is that we would only need this if the hyperlane contracts needed to query from the chain layer? But very low confidence

Reply:
IMO you are right. We use `stargate` queries on Osmosis for chain modules such as pool manager, twap, downtimedetector.